scripts/install_third_party.sh

```diff
@@ -136,3 +136,12 @@ if [ ! "$NO_JSREPL" -a ! -d "$THIRD_PARTY_DIR/static/jsrepl" ]; then
   mkdir -p $THIRD_PARTY_DIR/static/jsrepl
   cp -r $TOOLS_DIR/jsrepl/build/* $THIRD_PARTY_DIR/static/jsrepl
 fi
+
+echo Checking whether node-jscs dependencies are installed
+if [ ! -d "$NODE_JSCS_MODULE_DIR" ]; then
+  echo installing dependencies
+  cd $TOOLS_DIR
+  cd node-jscs
+  $NPM_INSTALL 
```

Comment:
Hm. Can we follow how the installation is done in run_js_tests.sh or run_integration_tests.sh instead -- i.e. specify the package name and a specific version?